lib/Agrammon/Web/Routes.pm6

```diff
@@ -272,10 +281,14 @@ sub application-routes(Agrammon::Web::Service $ws) {
         }
 
         ### data
-        # test
+        # implement branches
         post -> LoggedIn $user, 'store_data' {
             request-body -> %data {
-                my $ret = $ws.store-data($user, %data);
+                my $ret = $ws.store-data(
+                    $user,
+                    :dataset(%data<dataset_name>), :var(%data<data_var>), :value(%data<data_val>),
+                    :branches(%data<branches>), :options(%data<options>)
+                );
```

Comment:
Did you consider unpacking these in `request-body`, e.g. like:

```
request-body (Str :dataset_name($dataset)!, Str :data_var($var)!) { # and the others
    my $ret = $ws.store-data($user, :$dataset, :$var); # and the others
}
```

This means that the data will be validated and a 400 issued if it is mis-typed or mis-named (for example, an array isn't sent for `options` or `branches`). Whereas now it'll type-check fail on the call to `store-data` and that'll be a 500.